kernel/bitpattern.h

```diff
@@ -106,6 +135,14 @@ struct BitPatternPool
 		return false;
 	}
 
+	/**
+	 * Does sig match a pattern in the pool such that
```

Comment:
Can this be summarized as

> Are all combinations represented by the given signal contained in the set?

Is it true that `pool({aaa}).has_all(01a) == false`? That doesn't match what I think this function does.

Reply:
As a side note, the interface is a bit awkward in the sense that all the methods accept `SigSpec` but in fact they crash if the signal isn't convertable to `Const` (except for the constructor). Ideally we would change them to take `Const` instead.